src/services/index.js

```diff
@@ -3,5 +3,7 @@ export * from './auth.service';
 export * from './case.service';
 export * from './notification.service';
 export * from './chat.service';
+export * from './group.service';
+export * from './generalSearch.service';
```

Comment:
This should be renamed to `search.service` (applicable for references)

Reply:
Done Sir